### DISCUSSION TITLE
How to Customize y-axis values

### DISCUSSION THREAD

**Discussion by msafrun** [plouc/nivo#2380] [Q&A]
Can anyone help me on how to customize the y-axis value on any type of chart. What I would like to achieve is that the y-axis [4000, 8000, 12000] etc, but right now it shows [2000, 4000, 6000] I assume its multiply/modulo by 2.

please refer to this : [https://stackoverflow.com/questions/76586075/nivo-format-y-axis-for-any-chart-type](url)

**Comment by hyjoong** [comment]
You can specify the desired value in the tickValues property in AxisLeft.

```javascript
<ResponsiveLine
...
 axisBottom={{
        ...
        tickValues: [4000, 8000, 12000],
      }}
...
/>
```
